src/resources/js/components/EditPresentation.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+      <v-text-field
+            v-model="updatePresentationInfo.title"
+            :rules="titleRules"
+            label="Titre"
+      />
+
+      <v-text-field
+            v-model="updatePresentationInfo.date"
+            label="Date"
+      />
+
```

Comment:
missing actual values of the fields
How do I now what I'm modifying?

Reply:
Changed in 3687984